napari/utils/colormaps/colormap_utils.py

```diff
@@ -427,25 +481,30 @@ def label_colormap(num_colors=256, seed=0.5):
     """
     # Starting the control points slightly above 0 and below 1 is necessary
     # to ensure that the background pixel 0 is transparent
-    midpoints = np.linspace(0.00001, 1 - 0.00001, num_colors + 1)
-    control_points = np.concatenate(([0], midpoints, [1.0]))
+    midpoints = np.linspace(0.00001, 1 - 0.00001, num_colors - 1)
+    control_points = np.concatenate([[0], midpoints, [1.0]])
     # make sure to add an alpha channel to the colors
-    colors = np.concatenate(
-        (
-            _color_random(num_colors + 2, seed=seed),
-            np.full((num_colors + 2, 1), 1),
-        ),
-        axis=1,
-    )
-    # Insert alpha at layer 0
-    colors[0, :] = 0  # ensure alpha is 0 for label 0
+    if num_colors == 49:
+        colors = np.array(CLASSICAL_LABEL_COLORMAP)
+    else:
+        colors = np.concatenate(
```

Comment:
I think the intention was to *always* have the "classical" colormap for the first N (49) labels. So probably just concatenate `CLASSICAL_LABEL_COLORMAP` with the next `num_colors - 49` random colors? Not sure about this though, let's see what @jni meant.